### ISSUE TITLE
Django server isn't being started

### ISSUE THREAD

**Issue by andybak** [kitconcept/robotframework-djangolibrary#3]
There seems to be a hardcoded path to mysite/manage.py:

```
python: can't open file 'mysite/manage.py': [Errno 2] No such file or directory
```

**Comment by andybak** [comment]
Is there some way to make use of Django's own LiveServerTestCase to ensure we've got a running server?

**Comment by tisto** [comment]
Right. I need to make this configurable. I guess we could use LiveServerTestCase. So the use case is that you "pip install robotframework-djangolibrary" right into the directory of your Django instance, right?

**Comment by andybak** [comment]
pip installed into a virtualenv and running it from the same directory as my manage.py file

**Comment by tisto** [comment]
The issue should have been fixed in 1.0a3.
